components/resource_control/src/worker.rs

```diff
@@ -196,12 +217,12 @@ impl<R: ResourceStatsProvider> GroupQuotaAdjustWorker<R> {
         // the available resource for background tasks is defined as:
         // (total_resource_quota - foreground_task_used). foreground_task_used
         // resource is calculated by: (resource_current_total_used -
-        // background_consumed_total). We reserve 10% of the free resources for
+        // background_consumed_total). We reserve 20% of the free resources for
```

Comment:
why

Reply:
In my test, 10% is not enough to restrict the resource usage very well, this should due to our under calculation of the resource usage, so I adjust it to 20% and the result is much better in my benchmark.

Reply:
Rolled it back because I originally want to do this change in #15166